oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/storage/model/ModelName.java

```diff
@@ -30,10 +43,73 @@ public static String build(Downsampling downsampling, String modelName) {
                 return modelName + Const.ID_SPLIT + Downsampling.Day.getName();
             case Hour:
                 return modelName + Const.ID_SPLIT + Downsampling.Hour.getName();
-            //            case Second:
-            //                return modelName + Const.ID_SPLIT + Downsampling.Second.getName();
             default:
                 return modelName;
         }
     }
+
+    public static String[] build(Downsampling downsampling, String modelName, long startTB, long endTB) {
+        List<String> indexNameList = new ArrayList<>();
+
+        List<String> whiteIndexList = new ArrayList<String>() {
```

Comment:
This should be a static list, rather than create everytime.

Reply:
ok